tests/k8s/1-032_validate_dynamic_scaling/06-assert.yaml

```diff
@@ -0,0 +1,13 @@
+apiVersion: argoproj.io/v1alpha1
```

Comment:
Any specific reason why the file is name prefixed with `06` instead of `04` which is the next available integer in the sequence of steps ?

Reply:
Missed renaming the files. Thanks for catching that. Updated the file names.